Resources/doc/versioning.rst

```diff
@@ -77,17 +77,17 @@ automatically in the ``ViewHandler``.
 
 If you want to version by Accept header, you will need to do the following:
 
-#. The format listener must be enabled
+1. The format listener must be enabled
 
 See :doc:`Format Listener <format_listener>`
```

Comment:
We need to indent the content of the list items by three spaces each. That should fix the rendering of the list.

Reply:
Oh ok. I'll update this.